packages/coil-extension/src/background/services/AuthService.ts

```diff
@@ -194,12 +202,45 @@ export class AuthService extends EventEmitter {
     const resp = await this.client.queryToken(token)
     if (resp.data?.refreshToken?.token && resp.data?.whoami) {
       this.store.user = resp.data.whoami
+
+      // Data needed for tipping
+      // tipping-beta: featureEnabled: boolean
+      // minimum tip limit: minTipLimit > minTipLimit
+      // remaining daily amount: whoami > tipping > limitRemaining
+      this.formatTipSettings(resp.data.refreshToken.token) // adds the tip settings info to the user object and formats it
       return resp.data.refreshToken.token
     } else {
       return null
     }
   }
 
+  private async formatTipSettings(token: string) {
+    // convert all tip settings from cents to dollars
+    // set default hotkey tip amounts since we don't yet get them from the user
+    // add feature flag and minTipLimit
+    if (this.store.user) {
+      const featureFlagResp = await this.client.featureEnabled(
+        token,
+        'tipping-beta'
+      )
+      const minTipLimitResp = await this.client.minTipLimit(token)
+      const formattedTipSettings = {
+        inTippingBeta: featureFlagResp.data.featureEnabled,
+        minimumTipLimit: minTipLimitResp.data.minTipLimit.minTipLimit
```

Comment:
nit: it would be nice to break this logic out to a function, same for the lines below.